tests/foreman/ui_airgun/test_reporttemplates.py

```diff
@@ -396,7 +172,15 @@ def test_positive_e2e_crud(session, module_org, module_loc):
             'organizations.resources.assigned': [module_org.name],
             'locations.resources.assigned': [module_loc.name],
         })
-        assert session.reporttemplate.search(name)[0]['Name'] == name
+        # also check other properties by API once BZ1702676 gets fixed
+        rt = entities.ReportTemplate().search(
+                query={'search': 'name={0}'.format(name)})[0].read()
+        assert rt.name == name
+        assert rt.organization[0].read().name == module_org.name
+        assert rt.location[0].read().name == module_loc.name
+        assert rt.template == content
+        assert rt.default is True
+        # R
         pt = session.reporttemplate.read(name)
```

Comment:
Do you have a reason to keep `rt` around and introduce new variable?
I can see how `rt` stands for report template, but name `pt` is not very descriptive.

Same goes for `updated_pt` and `crt` below.